app/code/Magento/CustomerGraphQl/Model/Customer/Address/CreateCustomerAddress.php

```diff
@@ -67,6 +67,9 @@ public function __construct(
      */
     public function execute(int $customerId, array $data): AddressInterface
     {
+        if (isset($data['country_code'])) {
```

Comment:
Could you please add comment like `It is needed because AddressInterface has country_id field.`

Reply:
Done.